tests/ConditionSQLTest.php

```diff
@@ -244,6 +244,29 @@ public function testDateCondition()
         $this->assertEquals('Sue', $m['name']);
     }
 
+    public function testDateCondition2()
+    {
+        $a = [
+            'user' => [
+                1 => ['id' => 1, 'name' => 'John', 'date' => '1981-12-08'],
+                2 => ['id' => 2, 'name' => 'Sue', 'date' => '1982-12-08'],
+            ], ];
+        $this->setDB($a);
+
+        $m = new Model($this->db, 'user');
+        $m->addField('name');
+        $m->addField('date', ['type'=>'date']);
+
+        $m->addCondition('date', new \DateTime('08-12-1982'));
+        $m->loadAny();
+        $this->assertEquals('Sue', $m['name']);
+        $this->assertEquals('Sue', $m['name']);
```

Comment:
This line is a duplicate of the line before.

Reply:
yeah :) will fix